### DISCUSSION TITLE
using outdated @cts in new release

### DISCUSSION THREAD

**Discussion by 50NNY1** [ppooll-dev/ppooll#60] [User Feedback]
There are quite a few @cts missing, that are unstable and buggy - and fixed versions already have been made in the #act-share in the ppooll discord. Acts i've noticed already: forbiddenplannet (vocoder is broken), the new ll.blues is very problematic, changing send amounts from one act to another is very unpredictable, if you want to upload some of the best acts from the discord, i'd really reccomend taking a look at the efx acts they are wonderful and could use some love! Thank you once again ppooll, the elites of music production software!

**Comment by chausch** [comment]
dear @50NNY1 ,
i'm not sure i can follow exactly what your issues are. please try to describe the issues more precisely, and only one topic per issue thread!
please post general feedback in the discussions under "user feedback": 
https://github.com/ppooll-dev/ppooll/discussions/categories/user-feedback
(issues should be reserved for bug reports or feature requests).
anyway thanks for your words and using pppooll! 😊

**Comment by chausch** [comment]
ps: please don't misread it that i closed your issue – we're very happy to have feedback, it was just the wrong location for it and too general in it's wording. as i wrote above, please post it in discussions or write it to the mailing list!
regarding blues: if you have a specific issue with it please describe it clearly and open up a new issue!!

**Comment by chausch** [comment]
hi @50NNY1 – i just noticed it's possible to transfer an issue to a discussion. just did that with this thread!

**Comment by klausfilip** [comment]
hi @50NNY1 
sorry i only read this now.
as for any acts, you would like to include (old or new), please upload them into ppooll_contributions  !!
if you could do this, that would be great, none of us really can keep track of all the discord action.
